src/main/resources/application.conf

```diff
@@ -1,6 +1,6 @@
 api {
   host = "0.0.0.0"
-  port = 8080
+  port = 9999
```

Comment:
IMHO we should think about our port numbers so that one could remember them easier.

Also I have already a couple of commands with that port. Changing it means I will have to change my rest client/ commands.

So if this change is not really necessary I would like not to have it.

If this change is necessary we should think about sane port numbers across services.
It is getting wild, ie. ports already in use:
* 1024
* 3333
* 5555
* 4200

Reply:
Yes, 8080 is common; it is also the port used when we run our docs locally, so I think not using 8080 is probably more convenient.  
But I agree that we should streamline our ports. And since we have the API on `3333` and fuseki on `3030` it would probably make sense to go to the 3000-range, and maybe also change our ZIO-HTTP and our instrumentation port so something like this.

I would propose something along these lines:

```
1024 -> SIPI   # we don't touch this for now
3030 -> Fuseki # we could leave this or change it to 3330
3333 -> DSP-API
3334 -> DSP-API instrumentation
3335 -> DSP-API ZIO-HTTP # will eventually free up
# keep some ports free here
3340 -> DSP-Ingest
4200 -> DSP-APP # we could friendly ask the Frontend-Family if they want to move to port 3350, but this isn't our main concern
```

Reply:
Thanks @BalduinLandolt for exhausive explanation. `4200` is default port for Angular applications. We could change it too but does that hurt us to have all port in the same range? IMO it is harder to remember which is responsible for what. But in case of this Ingest service I will set is as you proposed on `3340`.